lib/docs/wasm/markdown/Parser.zig

```diff
@@ -285,8 +285,8 @@ pub fn feedLine(p: *Parser, line: []const u8) Allocator.Error!void {
                 try p.addScratchStringLine(rest_line_trimmed);
             }
 
-            if (maybe_containing_list) |containing_list| {
-                containing_list.data.list.last_line_blank = rest_line_trimmed.len == 0;
+            if (maybe_containing_list_index) |containing_list_index| {
+                p.pending_blocks.items[containing_list_index].data.list.last_line_blank = rest_line_trimmed.len == 0;
```

Comment:
i see you found this bug too. did you get it from my dev branch?

Reply:
One of my newly added test cases was initially segfaulting due to this, unfortunately 😅 I've been meaning to get a fuzzer set up for this implementation. It was very helpful when I worked on an XML parser a while back, and I wouldn't be surprised if there were still some interesting edge cases to be triggered here.

Reply:
oh, we both made exactly the same change then, haha. or actually, i named my capture `|i|` instead. :shrug: